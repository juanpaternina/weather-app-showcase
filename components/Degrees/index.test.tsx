import { render, screen } from '@testing-library/react-native';

const { Degrees } = require('.');

describe('Degrees', () => {
  test('It should render a Text component', () => {
    const { toJSON } = render(<Degrees degrees={10} />);
    expect(toJSON()).toMatchSnapshot();
    const text = screen.getByText('10');
    expect(text).toBeDefined();
  });
});
