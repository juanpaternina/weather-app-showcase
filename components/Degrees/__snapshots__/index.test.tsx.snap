// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Degrees It should render a Text component 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "alignSelf": "center",
      "flexDirection": "row",
      "paddingBottom": 10,
      "paddingLeft": 10,
      "paddingRight": 10,
      "paddingTop": 10,
    }
  }
>
  <Text
    style={
      [
        {
          "color": "black",
          "fontFamily": "bold",
          "fontSize": 24,
          "fontWeight": "bold",
        },
        {
          "fontSize": 150,
        },
      ]
    }
  >
    10
  </Text>
  <Text
    style={
      [
        {
          "color": "black",
          "fontFamily": "medium",
          "fontSize": 20,
          "fontWeight": "bold",
        },
        {
          "fontSize": 120,
          "marginTop": -50,
        },
      ]
    }
  >
    Â°
  </Text>
</View>
`;
